chat/tasks.py

```diff
@@ -45,9 +53,18 @@ def cleanup_puzzle_channels(puzzle_id):
                 logger.warn(f"cleanup_puzzle_channels failed with error: {e}")
 
 
+@shared_task(rate_limit="6/m")
```

Comment:
I actually couldn't figure out what the Discord rate limit is... According to https://discord.com/developers/docs/topics/rate-limits, <50 requests/second should be fine, but I'm hitting it locally before then. 

This is also actually not the best way to handle the rate limit either, because it only rate limits it for this one method and there are other discord calls elsewhere in this file. We're just most likely to hit it here since it does 2 calls (one for voice and one for text) and the UI makes it sorta easy to spam changing of metas. 

I'll add a TODO and point to #565 to fix it properly after this weekend.

Reply:
Gotcha, makes sense. What happens when we hit the rate limit, are we retrying? I know the sheets tasks have some retry with backoff & jitter.

Reply:
That is a good point. These should also retry, but I've never seen them retry, lemme figure out why not lol